community/modules/scripts/windows-startup-script/outputs.tf

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+output "windows_startup_ps1" {
+  description = "script to load and run all runners, as a string value."
+  value       = local.startup_ps1
```

Comment:
Would it be possible to output a list and then have the blueprint look like:
```
  - id: image
    source: modules/packer/custom-image
    kind: packer
    use: [network0]
    settings:
      windows_startup_ps1: $(windows_startup.windows_startup_ps1)
```

Reply:
With #1513 merged, I can have this do proper "use" syntax.